agent-inject/agent/config_test.go

```diff
@@ -29,7 +29,6 @@ func TestNewConfig(t *testing.T) {
 		AnnotationVaultClientKey:                        "client-key",
 		AnnotationVaultSecretVolumePath:                 "/vault/secrets",
 		AnnotationProxyAddress:                          "http://proxy:3128",
-		"vault.hashicorp.com/agent-inject-secret-foo":   "db/creds/foo",
```

Comment:
Same comment again - it would be good to preserve existing behaviour and just enable a new simplified set of annotations that previously wouldn't have worked.